src/components/modals/LargeModals.tsx

```diff
@@ -0,0 +1,115 @@
+import { StyleSheet, Text, View, Pressable, Image, Alert } from "react-native";
```

Comment:
@Allen-Muhani @Alphonce-Mutebi @Chegemartinx 
Another issue with aligning Figma and GitHub.

1. First, there is no modals folder on Figma. Just cards.
2. There is no LargeModals on Figma

Devs, if you need another folder, and or another file, like this modal, you ALWAYS have to circle back with Martin, to make sure that things are aligned. Otherwise nobody will ever understand what is going on and it will over time turn into a mess that will be impossible to clean up.

Again, not coding feedback, but still suuuuper important.
i know you are putting in a ton of work, and i think that is great. 
let us just make sure that we iron out these things, so we can focus all our energy on what is important, and not run in circles.

Reply:
This will be discussed in a meeting today after stand-up. Ill make the necessary changes then.